les/api_backend.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/celo-org/celo-blockchain/core/state"
 	"github.com/celo-org/celo-blockchain/core/types"
 	"github.com/celo-org/celo-blockchain/core/vm"
+	vmcontext "github.com/celo-org/celo-blockchain/core/vm/context"
```

Comment:
I would go for just calling this package vmcontext, duplicating package names can be confusing especially when those package names are from the standard library since clashes become even more likely.

Reply:
Yeah, had the same feeling. I always feel quite constrained by golang package names, since only the last part of the path matters...  and so you end up doing `vm/vmcontext` which duplicates the name, but makes it clear...

i'll do that change. @trianglesphere this should also solve your import comments